.github/CODEOWNERS

```diff
@@ -5,3 +5,4 @@
 **/utd_ganymede** @edmundmiller @alyssa-ab
 **/unsw_katana** @jscgh
 **/seadragon** @jiawku
+**/fred_hutch** @derrik-gratz
```

Comment:
Probably need to join the github org for this not to complain, see https://slack.com/shortcuts/Ft07NJUA4R0B/1549d7fef819a7d172c198f24ba3db11 (GitHub invitation request in #github-invitations in the nf-core slack if the link doesn't work).

Reply:
@jfy133 I don't recall, does something need to be done on the repo here to not trigger failure with derrik-gratz as codeowner?